### DISCUSSION TITLE
Could I change the label of author and date field?

### DISCUSSION THREAD

**Discussion by road2you** [quarto-dev/quarto-cli#600] [Q&A] [answered]
The `abstract-title` option is to change the label of `abstract` (HTML format). But I can't find any way to change the label of `author`/`date`.  Is there something I am missing?

I want to change the labels as follows :
`AUTHOR` -> `WRITER`
`PUBLISHED` -> `UPDATED`

**Comment by jjallaire** [comment, marked as answer]
Yes you can but when I checked there was no good documentation on this. I've remedied that here: https://quarto.org/docs/authoring/language.html#alternate-language

**Reply by road2you** [reply]
It works perfectly fine. Thank you. Thanks to the newly created document, I have a good understanding of the `language` option's meaning. Quarto Hurray!

**Comment by donboyd5** [comment]
When I experimented with this, it puts the new labels in upper case even if I write them in initial caps. Is there a way to make the case of the labels be the case as it is written? Thanks.

**Comment by jjallaire** [comment]
I think this probably needs to be handled at the CSS level. @dragonstyle Is there a straightforward way for users to write CSS to change this behavior?

**Comment by dragonstyle** [comment]
Right now, there is no built in option to customize this, so you have a couple of options:

1) You can style it yourself using CSS with something like

```css
.quarto-title-meta-heading {
  text-transform: none !important;
}
```

2. You could choose the default Pandoc style heading using `title-block-style: none` or to use our structure for the title block, but apply no built in styles so you can CSS to style and layout the title block as you'd like it use `title-block-style: plain`.

**Comment by donboyd5** [comment]
Thanks very much. I'll investigate this.  Don
